assets/dev/js/editor/elements/views/base.js

```diff
@@ -814,21 +813,36 @@ BaseElementView = BaseContainer.extend( {
 		return changed;
 	},
 
-	isCssIdControl( changedControl, bindingDynamicCssId ) {
-		return bindingDynamicCssId === changedControl;
+	*bindingChanges( changed, bindingSettings, config ) {
+		for ( const [ key, value ] of Object.entries( changed ) ) {
```

Comment:
I would use continue to reduce nesting
```suggestion
	for ( const [ key, value ] of Object.entries( changed ) ) {
		if ( '__dynamic__' !== key && ! this.isHandledAsDatabinding( key, bindingSettings, config ) ) {
			continue;
		}

		if ( '__dynamic__' !== key ) {
			yield { key, value, isDynamic: false };
			continue;
		}

		for ( const [ dynamicKey, dynamicValue ] of Object.entries( value ) ) {
			if ( this.isHandledAsDatabinding( dynamicKey, bindingSettings, config ) ) {
				yield { key: dynamicKey, value: dynamicValue, isDynamic: true };
			}
		}
```